cumulus/tasks/pdr-status-check/index.js

```diff
@@ -130,6 +131,21 @@ function buildOutput(event, groupedExecutions) {
   return output;
 }
 
+/**
+ * check the status of a step funciton execution
+ *
+ * @param {string} executionArn - step function execution arn
+ * @returns {Promise.<Object>} - an object describing the status of the exection
+ */
+function describeExecutionStatus(executionArn) {
```

Comment:
Not sure if this matters but this is kind of duplicating functionality we have in https://github.com/cumulus-nasa/cumulus/blob/master/packages/api/endpoints/execution-status.js - we could think about putting this in `@cumulus/common/aws` and sharing code between these 2 modules (`cumulus/tasks/pdr-status-check/index.js` and `packages/api/endpoints/execution-status.js`) from `@cumulus/common/aws`

Reply:
But it might be we want different enough behavior that keeping them separate makes sense

Reply:
It looks similar, but combining them doesn't seem save much code.